heat/core/dndarray.py

```diff
@@ -2272,6 +2272,56 @@ def __or__(self, other):
         """
         return arithmetics.bitwise_or(self, other)
 
+    def pad(self, pad, mode="constant", value=0):
+        """
+        Pads tensor with a specific value (default=0).
+        (Not all dimensions supported)
+
+        Parameters
+        ----------
+        self : ht.DNDarray
```

Comment:
Adjust to 504 docstring formatting

Reply:
Done in manipulations.py (or at least I tried).